db/hints/manager.hh

```diff
@@ -97,21 +98,21 @@ public:
 
             enum class send_state {
                 segment_replay_failed,  // current segment sending failed
```

Comment:
Since there's only one state now, can we change the field that has type send_state_set to bool?

Reply:
Done, addressed in https://github.com/scylladb/scylla/pull/6416